distribution/sdk/artemis-sandbox/pom.xml

```diff
@@ -55,7 +55,45 @@
             <artifactId>artemis-jms-client</artifactId>
             <version>${artemis.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This dependency change seems unrelated to the other changes. Just wanted to make sure it is needed.